Resources/Private/Templates/Posting/List.html

```diff
@@ -107,9 +107,9 @@ <h3>
 	</div>
 	<script>
 		const options = {
-			plugins : ['remove_button']
+			plugins : ['remove_button'];
```

Comment:
This seems to be wrong, there can't be a semicolon in an object. It should probably be a line lower?

Reply:
Hi @itx-benjamin-jasper,
I have concatenateJs and compressJs in productive mode on and then the script does not work without the changes.
However, I am not sure which of the settings leads to the problem, but it doesn't matter.
You can test it in all current TYPO3 versions.

```
config {
   concatenateJs = 1
   compressJs = 1
}
```

https://docs.typo3.org/m/typo3/reference-typoscript/main/en-us/Setup/Config/Index.html

Reply:
Alright, interesting, as this sounds like something others could encounter as well, I am accepting your PR, but could you move the semicolon, as I asked earlier?
> This seems to be wrong, there can't be a semicolon in an object. It should probably be a line lower?